app/connectors/DeparturesMovementsP5Connector.scala

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package connectors
+
+import config.FrontendAppConfig
+import logging.Logging
+import models.{DeparturesSummary, DraftAvailability}
+import uk.gov.hmrc.http.HttpReads.Implicits._
+import uk.gov.hmrc.http.HttpReads.is2xx
+import uk.gov.hmrc.http.{HeaderCarrier, HttpClient, HttpResponse}
+
+import javax.inject.Inject
+import scala.concurrent.{ExecutionContext, Future}
+
+class DeparturesMovementsP5Connector @Inject() (config: FrontendAppConfig, http: HttpClient)(implicit ec: ExecutionContext) extends Logging {
+
+  def getDeparturesSummary(queryParams: Seq[(String, String)] = Seq.empty)(implicit hc: HeaderCarrier): Future[Option[DeparturesSummary]] = {
+    val url = s"${config.draftDeparturesUrl}/user-answers"
+
+    http.GET[HttpResponse](url, queryParams).map {
+      case response if is2xx(response.status) =>
+        response.json.asOpt[DeparturesSummary]
+      case response if response.status == 404 =>
+        Some(DeparturesSummary(List.empty))
+      case _ =>
+        None
```

Comment:
Do we want to log something here?

Reply:
I think we are going to update the way this is done once we move onto the pagination anyways. As we have a count now in the cache I think we will return an empty UserAnswers with the count set to zero rather than a 404. Means we can remove the need to match on the http response